apps/theme/layouts/forms/DefaultFormLayout.tsx

```diff
@@ -40,7 +63,35 @@ const DefaultFormLayout: FormLayoutComponent = ({
     const [formSuccess, setFormSuccess] = useState(false);
 
     // All form fields - an array of rows where each row is an array that contain fields.
-    const fields = getFields();
+    const fields = getFields(currentStep);
+
+    // Check if the form is a multi step.
+    const isMultiStepForm = formData.steps.length > 1;
+
+    // Validate fields for current step with "form.validateInput" function,
+    // if current step is invalid then we should block posibility to move to the next step,
+    // but if step is valid then user can switch to the next step.
+    const validateCurrentStepFields = (form: FormAPI) => {
```

Comment:
Let's also have this as an internal function users can immediately use.

But one thing though... When you call `submit` (the one passed by the `Form` component, L122), the rendered fields will be automatically validated. And then the `Form` component only calls your `onSubmit` callback if the validation is successful.

In other words, the validation of fields should probably be:

1) handled by the @webiny/app-form-builder's Form component internally.
2) should be internally handled by the `submit` callback / the @webiny/form package which triggers it automatically for us.

Hope I didn't miss anything here.